### ISSUE TITLE
Crafting menu, global filter by difficulty

### ISSUE THREAD

**Issue by Shoes01** [CleverRaven/Cataclysm-DDA#19327]
Sometimes there are so many things to craft that it's hard to parse all the information. In order to help limit the amount of information seen, I am thinking about adding a global filter that would only show recipes of difficulty X or more. The number X would be changed by using page up or page down in the crafting screen.

Is this something that is worth developing, or am I bloating the code without solving the actual problem?

**Comment by Coolthulhu** [comment]
Sounds relatively simple, as long as you wouldn't want it to understand multiple skills and only the difficulty stat.
Could be useful for finding the recipes to grind with.

**Comment by Shoes01** [comment]
Yup, exactly. Difficulty only. Cool; I'll see what I can do!

**Comment by pisskop** [comment]
I this sounds like a tool to assist grinding more than one to see what cool new recipes youve unlocked

**Comment by Shoes01** [comment]
I am not sure what you're saying :) Are you hoping to have the ability to see "recently unlocked" recipes? I imagine it wouldn't be a big stretch to have "new" recipes show up in a different color until they have been viewed.

**Comment by pisskop** [comment]
Nothing wrong with it, but Im just calling it as i see it.

**Comment by Shoes01** [comment]
Given the effort to lower the issue count, I will close this discussion now that I have my answer.
